packages/react-components/react-combobox/stories/Combobox/ComboboxHighlightedOptionChange.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { Combobox, makeStyles, Option, shorthands, useId } from '@fluentui/react-components';
+import type { ComboboxProps } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  root: {
+    // Stack the label above the field with a gap
+    display: 'grid',
+    gridTemplateRows: 'repeat(1fr)',
+    justifyItems: 'start',
+    ...shorthands.gap('2px'),
+    maxWidth: '400px',
+  },
+});
+
+export const HighlightedOptionChange = (props: Partial<ComboboxProps>) => {
+  const comboId = useId('combo-highlighted-option-change');
+  const options = ['Cat', 'Dog', 'Ferret', 'Fish', 'Hamster', 'Snake'];
+  const styles = useStyles();
+
+  const [activeOptionText, setActiveOptionText] = React.useState('');
+
+  const eventCount = React.useRef(0);
+
+  const onHighlightedOptionChange = React.useCallback(
+    (e, data) => {
+      eventCount.current++;
+      setActiveOptionText(
+        `HighlightedOptionChange: ${data?.nextOption?.text} Called ${eventCount.current} times.`,
```

Comment:
I think a better example would be to just display what the active option, it's clearer since that will correspond directly to what the user will be testing in the example

Reply:
yes, this is for debugging purposes only, to make sure we only call the callback when necessary. I will simplify the example.